packages/mdc-theme/_variables.scss

```diff
@@ -129,6 +129,10 @@ $mdc-theme-property-values: (
 //
 // NOTE: This function must be defined in _variables.scss instead of _functions.scss to avoid circular imports.
 @function mdc-theme-prop-value($style) {
+  @if mdc-theme-is-var-with-fallback_($style) {
```

Comment:
Returning the new style type as is causes an issue to other mixins that uses `mdc-theme-prop-value()` function. For example, `mdc-card-outline` uses this function to set border-color. [mdc-card/_mixins.scss:L303](https://github.com/material-components/material-components-web/blob/b9c5fc6c684d82f63d007761214edf3ca21f99d4/packages/mdc-card/_mixins.scss#L303)

Reply:
Can we update that line to be as follows?

```
@include mdc-theme-prop(border-color, $color);
```

Reply:
List divider uses the approach described above. I'll include that update in this PR.

https://github.com/material-components/material-components-web/blob/b9c5fc6c684d82f63d007761214edf3ca21f99d4/packages/mdc-list/_mixins.scss#L360

Reply:
I'm confused though, what issue was this causing? Do we need to be concerned for existing valid cases of mdc-theme-prop-value being used?

Reply:
It is still an issue if component needs to mix colors or apply opacity using `rgba()` sass function.

For example, dialog uses `rgba()` to apply opacity to color.

**dialog/_mixins.scss:**
```
background-color: rgba(mdc-theme-prop-value($color), $opacity);
```
Where, `mdc-theme-prop-value` returns new color format which rgba function doesn't support.

This is another advanced use case where text field uses `mix()` function two mix two colors:

**textfield/_variables.scss:**
```
$mdc-text-field-background:
    mix(mdc-theme-prop-value(on-surface), mdc-theme-prop-value(surface), 4%);
```

Reply:
@kfranqueiro It shouldn't break existing valid usage. This is an issue only for users who send new color format to component's mixins.

```
mdc-dialog-title-ink-color('purple'); // works
mdc-dialog-title-ink-color({
  varname: '--custom-css-var-ink-color',
  fallback: purple
}); // breaks
```

Reply:
`mdc-theme-prop-value` has been updated to always return the fallback value.

https://github.com/material-components/material-components-web/blob/e58979ecd8a1cac81f1cbf2d3501d5ce19b76d20/packages/mdc-theme/_variables.scss#L131-L138